.github/workflows/docs.yaml

```diff
@@ -11,13 +19,30 @@ jobs:
         with:
           # this fetches all branches. Needed because we need gh-pages branch for deploy to work
           fetch-depth: 0
-      - uses: actions/setup-python@v2
+      - uses: ruby/setup-ruby@v1.160.0
         with:
-          python-version: '3.9'
-      - name: Install MkDocs
-        run: pip install -r docs/doc_requirements.txt
-      - name: Publish docs
+          ruby-version: '3.2'
+
+      - name: Install Asciidoc
+        run: make docs-dependencies
+      - name: Build docs
+        run: make docs
+
+      - name: Prepare gh-pages branch
         run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
-          mkdocs gh-deploy --force --clean --verbose
+
+          git branch -D gh-pages &>/dev/null || true
+          git checkout -b gh-pages 95fa4ca0ec7a38ee8dc0aabcd39cbfa98745d3d4
```

Comment:
This command will be removed/adapted after a successfull check?

Reply:
the idea is to get only static html stuff on gh-pages branch, I'm copying the adoption logic: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/.github/workflows/docs.yaml#L38

Reply:
ack!